civihr_default_theme/templates/page/page.tpl.php

```diff
@@ -56,6 +68,17 @@
       </ul>
     </div>
     <?php } ?>
+    <?php if (user_access("administer users") && user_access("administer nodes") ) { ?>
+    <div class="chr_header__settings-menu">
+      <span class="icon-settings">
+        <i class="fa fa-cog" aria-hidden="true"></i>
+      </span>
+      <ul class="chr_header__settings-menu__sub-menu">
```

Comment:
I prefer to properly use BEM. Please refer to this: http://getbem.com/faq/#css-nested-elements